testing/pom.xml

```diff
@@ -361,6 +361,10 @@
                   <goal>verify</goal>
                 </goals>
                 <configuration>
+                  <systemPropertyVariables>
+                    <ccm.version>3.11.14</ccm.version>
+                    <stargate.test.nodes>1</stargate.test.nodes>
+                  </systemPropertyVariables>
```

Comment:
why is this added, we did not have this before?

Reply:
Both OSS C* 4.0.x and DSE 6.8.x had this and I added it here for OSS C* to make sure it is consistent with other versions thinking it might have been missing in the first place.

Reply:
> The builder image is also updated on the v1 branch, thus you need to bump version again, to v1.0.15

GCR image version on `v1` branch prior to #2359 was at `1.0.14` and on `main` branch prior to this PR at `1.0.13`. Should we keep the images versions on both brances at the same version number? In that case, was that missed on the `main` branch earlier?

> Are you able to build and push the new image?

I didn't have access to GCB and hence Jeff was able to help with that in testing #2359 yesterday.